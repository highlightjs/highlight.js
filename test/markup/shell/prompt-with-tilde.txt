~/docs> cat readme.adoc
= Highlight.js

Highlight.js is a syntax highlighter written in JavaScript.
