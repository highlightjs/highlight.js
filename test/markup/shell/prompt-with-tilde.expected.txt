<span class="hljs-meta">~/docs&gt;</span><span class="bash"> cat readme.adoc</span>
= Highlight.js

Highlight.js is a syntax highlighter written in JavaScript.
